php/content.md

```diff
@@ -65,6 +65,27 @@ same commands to build and run:
 Where `src/` is the directory containing all your php code and `config/`
 contains your `php.ini` file.
 
+### How to install more PHP extensions
+
+We provide a convenient script named `docker-php-ext-install`, you can use it to
+easily install PHP extension.
+
+For example, if you want to have a PHP container with `gd` and `mcrypt` 
+extensions, you can write your own Dockerfile like this:
+
+    FROM php:5.5.19-fpm
+    # Install modules
+    RUN apt-get update && apt-get install -y \
+        apt-utils re2c g++ \
+        zlib1g zlib1g-dbg zlib1g-dev zlibc \
+        libpng12-0 libpng12-dev libpng3 \
+        libjpeg9 libjpeg9-dbg libjpeg9-dev \
+        libmcrypt-dev libmcrypt4 mcrypt \
+        && docker-php-ext-install gd mcrypt
+    CMD ["php-fpm"]
```

Comment:
The example is using php 5.5, which is probably more common than @tianon's example with 5.6 (since it is still quite new).

We should minimally change to `php:5.5-fpm` so that it does not have to be updated quite as often, rather than the `5.5.19-fpm` which is already out of date.

Also, `libjpeg9` (plus dev and dbg versions) does not exist in `debian:jessie`; it has been replaced by `libjpeg62-turbo`.  Does it really need all those dependencies to build and later run?  Can we simplify it like @tianon's example?